src/app/trainee/mock-data/trainee-spec-data.ts

```diff
@@ -0,0 +1,20 @@
+// NOTE only import into tests or mock. Must remove from all imports from  production scripts
+export const RevalidationHistoryRespone1 = {};
+export const RevalidationHistoryRespone2 = JSON.parse(
```

Comment:
is `RevalidationHistoryRespone2` still needed or is it some data to be part of the same api endpoint `revalidation/api/recommendation/{gmcId}`?

i can appreciate the notes api is not ready so thats fine

Reply:
Yeah notes and history 'spec-data' is used for only spec-tests. will reuse in resolver tests as soon as next ticket scenario says redirect. Never imported into code-base because spec files are excluded.

![Screenshot 2020-04-24 at 14 13 29](https://user-images.githubusercontent.com/13540819/80216471-02b51800-8636-11ea-8777-cf04557878aa.png)

Reply:
thats fair